### DISCUSSION TITLE
I have an approved Education Plan, But still on Free Plan

### DISCUSSION THREAD

**Discussion by ek8819** [community/community#148079] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi I have an Approved Application on GitHub Education and it was Submitted Dec 24, 2024, and on GitHub Education Page I have this message"
Congrats! Your request for [@ek8819](https://github.com/ek8819) was approved and your benefits are now available.", But for now when Opening Copilot Page from GitHub Website or see plan from VS Code I still see that I am on Free Plan
Please, Help me for fixing this issue
Thanks on advance

**Comment by davevad93** [comment]
Hi @ek8819  ,

Check if you can redeem your offer by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
